### ISSUE TITLE
Migrate to modern property pages

### ISSUE THREAD

**Issue by AdamYoblick** [microsoft/PTVS#8233]
Create a python application in VS. In solution explorer, right click on the Project and click on Properties. You will see something like this:

![Image](https://github.com/user-attachments/assets/08b1b17e-1932-411e-91b6-fae8bcb6cfe2)

This is a very old UI experience that violates a few Microsoft accessibility tenets (it doesn't support Dark Mode theme, for example), and has a variety of other problems.

We are going to move up to the new property pages. Docs for this are at https://github.com/dotnet/project-system/tree/main/docs/repo/property-pages
